protocols/imap/src/test/java/org/apache/james/imap/processor/base/UidMsnMapperTest.java

```diff
@@ -218,6 +218,45 @@ public void removeShouldKeepAValidMappingWhenDeletingMiddle() {
     }
 
     @Test
+    public void addUidShouldCommute() {
```

Comment:
Commute means order of operation don't matter.

It's undergate  mathematic degree as far as I remember:

```
a°b = b°a
```

Here

```
addUid(1)
addUid(2)
```

Is equivalent to

```
addUid(2)
addUid(1)
```